System.Compression.Primitives/System/IO/Compression/Brotli/BrotliStream.cs

```diff
@@ -10,20 +10,18 @@
 #else
 using nuint = System.UInt32;
 #endif 
-namespace System.IO.Compression.Brotli
+namespace System.IO.Compression
 {
     public partial class BrotliStream : Stream
```

Comment:
I think we are doing it backwards. We should first have flat Span<byte> based APIs and then wrap them in Stream.